File: ProjetoFinal/Eventos/src/main/java/com/leonardoFanck/Eventos/Model/Certificado.java
package com.leonardoFanck.Eventos.Model;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import java.time.LocalDateTime;
import java.util.UUID;

@Getter
@Setter
@Entity
@AllArgsConstructor
@NoArgsConstructor
@Table(name = "certificado")
public class Certificado {

    @Id
    @GeneratedValue
    private UUID id;

    @Column(name = "user_id")
    private UUID userId;
    private UUID eventoId;

    @Column(name = "data_emissao")
    private LocalDateTime dataEmissao;

    @Column(name = "codigo_autenticacao")
    private String codigoAutenticacao;

    @Lob
    private byte[] pdfArquivo;

}

File: ProjetoFinal/EventosAuth/EventosShared/Model/Certificado.cs
namespace EventosShared.Model;

public class Certificado
{
	public Guid Id { get; set; }
	public Guid UserId { get; set; }
	public Guid EventoId { get; set; }

	public DateTime DataEmissao { get; set; }
	public string CodigoAutenticacao { get; set; } = string.Empty;

	public byte[] PdfArquivo { get; set; } = [];
}
